tests/t402-qfunction.c

```diff
@@ -24,9 +24,16 @@ int main(int argc, char **argv) {
   CeedQFunctionView(qf_mass, stdout);
 
   CeedQFunctionContextCreate(ceed, &ctx);
-  CeedScalar ctxData[5] = {1, 2, 3, 4, 5};
-  CeedQFunctionContextSetData(ctx, CEED_MEM_HOST, CEED_COPY_VALUES,
-                              sizeof(ctxData), &ctxData);
+  if (CEED_SCALAR_TYPE == CEED_SCALAR_FP64) {
+    CeedScalar ctxData[5] = {1, 2, 3, 4, 5};
+    CeedQFunctionContextSetData(ctx, CEED_MEM_HOST, CEED_COPY_VALUES,
+                                sizeof(ctxData), &ctxData);
+  }
+  else { // Make context twice as long so the size is the same in output
```

Comment:
`} else {`

Reply:
This was fixed in b628dc116a24e09db24d9f0fa78c6f57e5018f9d